src/oaklib/implementations/gilda.py

```diff
@@ -48,8 +48,11 @@ def __post_init__(self):
             # when parsed from a descriptor like ``gilda:<path>` via
             # :func:`get_resource_from_shorthand`. If no <path> was
             # given, then this will default to the default Gilda index
-            self.grounder = Grounder(terms=self.resource.slug)
-
+            try:
+                self.grounder = Grounder(terms=self.resource.slug)
```

Comment:
is it possible to have a resource where the slug attribute isn't available?

Reply:
Not really an issue with gilda or the grounder, just with how it's being called. It's OK if terms=None, but it doesn't work if self.resource=None since there's no slug. That only breaks the parent classes if you call `annotate_text` on them or something. Shouldn't happen through the usual CLI usage of course.